.github/workflows/verify_samples.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Samples
+
+on: 
+  push:
+    branches: master
+  pull_request:
+    branches: '*'
+
+jobs:
+  linux:
+    name: Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: |
+          sudo apt-get update -y && sudo apt-get install -y flex bison clang lld
+      - name: Build
+        run: sh ./.ci_build_samples.sh
+  macOS:
+    name: macOS
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: recursive
+      - name: Install Dependencies
+        run: |
+          brew update && brew install llvm
```

Comment:
I don't know how much of a problem this is or will be, but on Travis, which uses quite old images, brew cleanup gets triggered every time. I considered PRing a change to fix this and shave a bit off of our build time, but I expect this to replace Travis.

If you want to add the change to be on the safe side when it comes to minimizing build times, this should do the trick:
`export HOMEBREW_NO_INSTALL_CLEANUP=1`

Reply:
Fixed.